docs/_rule_template.md

```diff
@@ -1,22 +1,41 @@
+The following provides a template with general instructions for creating an Rule Definition Strategy (RDS) document for defining rules in the Ruleset Checking Tool.  
+
 # <SECTION_NAME> - Rule <RULE_ID>
-**Rule ID:** <RULE_ID>  
-**Rule Description:** <RULE_DESCRIPTION>  
-**Rule Assertion:** <RULE_ASSERTION>  
-**Appendix G Section:** <SECTION_NAME>  
+**Rule ID:** The unique string identifier used to identify the rule within the ruleset.  
+**Rule Description:** A textual description description of the rule intended to be human readable. This description is included in the output JSON created by the RCT.  
+**Rule Assertion:** The logical assertion that is applied to determine the outcome of a rule evaluation. Options   
+**Appendix G Section:** The section number reference to the rule within Standard 90.1. e.g., *Section G3.1-5(b) Building Envelope Modeling Requirements for the Baseline building.*    
 **Appendix G Section Reference:**
-- <APPENDIX_G_SECTION_NUMBER>  
+- A list of references to other ruleset sections used within the rule.
+- Use *None* if no references are required. 
 
-**Data Lookup:** <APPENDIX_G_TABLE_NUMBER>  
-**Evaluation Context:**  <"Each Data Element" or "Data Group">  
+**Data Lookup:** A reference to specific data tables within ASHRAE 90.1 that are required for the rule evaluation. e.g., *Tables G3.4-1 to G3.4-8*  
+**Evaluation Context:** A string signifying at what level with the RMR data tree the rule evaluation must occur. Options are *Each Data Element* and *Data Group*. 
+- A rule that is evaluated at *Each Data Element* requires iteration over each data element that exists within a data group in the RMR. A rule assertion is then applied independently for each applicable data element. 
+- A rule that is evaluated at a *Data Group* is evaluated once for the relevant data group. This type of evaluation usually involves aggregating or calculating values up to the data group level prior to applying the assertion.  
 
 **Applicability Checks:** 
-1. <APPLICABILITY_CHECK_DESCRIPTION>
+1. This is a list of all checks that are performed to determine whether the rule is applicable to elements of the RMR. The method for conducting applicability checks is related to the *Evaluation Context*.  
```

Comment:
I'm not sure what the numbering "1." is intended to signify.  Perhaps remove it?

Similarly in the **Function Call:** section.